diff -Nur mediawiki-1.11.0.old/AdminSettings.php mediawiki-1.11.0/AdminSettings.php
--- mediawiki-1.11.0.old/AdminSettings.php	1969-12-31 19:00:00.000000000 -0500
+++ mediawiki-1.11.0/AdminSettings.php	2007-10-09 17:32:52.000000000 -0400
@@ -0,0 +1,20 @@
+<?php
+/**
+ * On scripts.mit.edu, the user's sole account can be used
+ * both for regular operation of the wiki and for any
+ * maintenance.  Therefore, we just use the username and
+ * password already set in LocalSettings.php.
+ *
+ * @package MediaWiki
+ */
+          
+$wgDBadminuser      = $wgDBuser;
+$wgDBadminpassword  = $wgDBpassword;
+          
+/*
+ * Whether to enable the profileinfo.php script.
+ * (False is the default.)
+ */
+$wgEnableProfileInfo = false;
+          
+?>
